src/main/java/org/sunbird/common/executor/SchedulerTrigger.java

```diff
@@ -9,18 +9,22 @@
 import org.sunbird.common.helper.cassandra.CassandraConnectionMngrFactory;
 import org.sunbird.common.util.Constants;
 import org.sunbird.common.util.KeyManager;
+import org.sunbird.common.util.PropertiesCache;
 import org.sunbird.scheduler.util.SchedulerManager;
 
+import static org.sunbird.common.util.Constants.SEND_NOTIFICATION_PROPERTIES;
+
 @Component
 public class SchedulerTrigger implements ApplicationRunner {
```

Comment:
Clean up the imports in this class and just call createConnection directly instead of creating an object